### PR TITLE
11.0 mig sale commercial partner

### PR DESCRIPTION
**Pull request by Swapnesh-SerpentCS**

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
Squash last 4 commits @Swapnesh-SerpentCS

**Comment by pedrobaeza** [comment]
Please squash latest commits

**Review by atchuthan — APPROVED** [review]
Code review :+1:

**Comment by Swapnesh-SerpentCS** [comment]
Done @pedrobaeza

**Comment by Swapnesh-SerpentCS** [comment]
Kept dependency on `sale` module itself As no part of `sale_managent` is used here, But functionally It should be dependent on `sale_management`
What is your view on this? @pedrobaeza

**Comment by pedrobaeza** [comment]
We can keep the dependency minimal in this case. Please add a final extra empty line in several files that you don't have it.

**Comment by Swapnesh-SerpentCS** [comment]
Added (Was needed in one file only)

File: sale_commercial_partner/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
```
Comment (pedrobaeza) [inline sale_commercial_partner/__init__.py]: Missing extra line

Comment (Swapnesh-SerpentCS) [reply sale_commercial_partner/__init__.py]: Added

**Review by pedrobaeza — APPROVED** [review]

**Review by alexis-via — APPROVED** [review]

**Pull request merged**